lynxkite-graph-analytics/src/lynxkite_graph_analytics/lynxkite_ops.py

```diff
@@ -75,6 +75,45 @@ def import_file(
     return core.Bundle(dfs={table_name: df})
 
 
+@op(
+    "Export to File",
+)
+def export_to_file(
+    bundle: core.Bundle,
+    *,
+    filename: str,
+    file_format: FileFormat = FileFormat.csv,
+    columns: str = "<all>",
+    index: bool = False,
+):
+    """Exports a DataFrame to a file.
+
+    Args:
+        bundle: The bundle containing the DataFrame to export.
+        filename: The name of the file to export to.
+        file_format: The format of the file to export to. Defaults to CSV.
+        columns: Comma-separated list of columns to export.
+            Defaults to "<all>" which exports all columns.
+        index: Whether to include the DataFrame index in the exported file. Defaults to False.
+    """
+
+    df = bundle.dfs["result"]
+    if columns == "<all>":
+        columns = None
+    else:
+        columns = [col.strip() for col in columns.split(",")]
+    if file_format == FileFormat.csv:
+        df.to_csv(filename, index=index, columns=columns)
+    elif file_format == FileFormat.json:
+        df.to_json(filename, orient="records", lines=True)
+    elif file_format == FileFormat.parquet:
+        df.to_parquet(filename, index=index)
+    elif file_format == FileFormat.excel:
+        df.to_excel(filename, index=index)
```

Comment:
These formats ignore `columns`. That's a bit error prone -- you export a CSV without the secret stuff, it's all good, then you later change to JSON and suddenly the secrets are in the JSON.

- One option is to apply `columns` everywhere. This is not too hard. You can just do `df = df[columns]`.
- But it's even easier to just drop the `columns` parameter. The user can already remove columns easily.